vault/vault.go

```diff
@@ -52,12 +56,15 @@ type vaultSecrets struct {
 	isKvBackendV2 bool
 	autoAuth      bool
 	config        map[string]interface{}
+	cooldown      time.Time
 }
 
 // These variables are helpful in testing to stub method call from packages
 var (
-	newVaultClient = api.NewClient
-	isKvV2         = isKvBackendV2
+	newVaultClient     = api.NewClient
+	isKvV2             = isKvBackendV2
+	errInCooldown      = errors.New("vault client is in cooldown")
```

Comment:
Define this as a global vault package error?

Reply:
fixed in [148196b](https://github.com/libopenstorage/secrets/pull/86/commits/148196bffcdc2a86436036cb65868b72f3248ebb)